### PR TITLE
Fix failing tests in `test/celery_tasks_test.py`.

### LINKED ISSUE #1623: Fix failing tests in `tests/celery_tasks_test.py`.
##### General

- [x] Make unit tests idempotent.

##### Fix the following test cases

- [x] `test_cleanup` (`celery_tasks_test.TestCleanup`)
- `test_result` (`celery_tasks_test.TestUpgradeScreenly`) &ndash; won't fix, `upgrade_screenly` task is not defined
- `test_state` (`celery_tasks_test.TestUpgradeScreenly`) &ndash; won't fix, `upgrade_screenly` task is not defined
- [x] `test_append_usb_assets` (`celery_tasks_test.TestUsbAssets`)
- [x] `test_cleanup_usb_assets` (`celery_tasks_test.TestUsbAssets`)
- [x] `test_remove_usb_assets` (`celery_tasks_test.TestUsbAssets`)

#### Overview of the Issue

```
test_cleanup (celery_tasks_test.TestCleanup) ... ERROR
test_result (celery_tasks_test.TestUpgradeScreenly) ... ERROR
test_state (celery_tasks_test.TestUpgradeScreenly) ... ERROR
test_append_usb_assets (celery_tasks_test.TestUsbAssets) ... /usr/src/app/server.py:139: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  usb_file_settings = yaml.load(yaml_file).get('screenly')
ERROR
test_cleanup_usb_assets (celery_tasks_test.TestUsbAssets) ... ERROR
test_remove_usb_assets (celery_tasks_test.TestUsbAssets) ... ERROR

...

Ran 6 tests in 0.231s

FAILED (errors=6)
```

##### First Time Setup

```bash
docker-compose -f docker-compose.dev.yml build
docker-compose -f docker-compose.dev.yml up -d
docker-compose -f docker-compose.dev.yml exec srly-ose-test bash bin/prepare_test_environment.sh -s
```

##### Running the Tests

```bash
docker-compose -f docker-compose.dev.yml exec srly-ose-test nosetests -v -a 'fixme' tests/celery_tasks_test.py
```

#### Environment

- WSL 2 (Ubuntu 22.04, x86, 64-bit)
- Docker Desktop for Windows (accessible via WSL 2)

#### Logs

- [issue_1623.log](https://github.com/Screenly/screenly-ose/files/9716459/issue_1623.log)

### PR DESCRIPTION
**Pull request by nicomiguelino**
Might fix #1623.

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -21,6 +21,8 @@ services:
     tty: true
     volumes:
       - .:/usr/src/app
+    environment:
+      - LISTEN=0.0.0.0
```
Comment (nicomiguelino) [inline docker-compose.dev.yml:25]: If the container is solely for running unit tests, then these lines could be removed.

File: tests/celery_tasks_test.py [hunk 2]
```diff
@@ -16,8 +16,12 @@
 
 class CeleryTasksTestCase(unittest.TestCase):
     def setUp(self):
+        self.image_url = 'https://www.screenly.io/upload/ose-logo.png'
```
Comment (vpetersson) [inline tests/celery_tasks_test.py:19]: Let's remove this as I don't think it exists anymore. Perhaps use something from the GitHub repo instead.

**Review by vpetersson — CHANGES_REQUESTED** [review]
Lgtm with one minor change.

File: tests/celery_tasks_test.py (hunk 2 above)
Comment (nicomiguelino) [reply tests/celery_tasks_test.py:19]: Alright, thanks! Will have to use the following instead: https://github.com/Screenly/screenly-ose/raw/master/static/img/ose-logo.png

**Pull request merged**